docs/network/networks/mainnet.md

```diff
@@ -13,7 +13,9 @@ You can set your Lit Network in the [LitNodeClient config of the Lit SDK](../../
 | Name | Description | Supported Algorithms | Supported Features | Status | SDK Version | Development status | Contracts |
 | ---- | ----------- | -------------------- | ------------------ | ------ | ----------- | -------------------- | --------------- |
 | Jalapeno | Centralized alpha network. Persistent, keys will not be deleted. | BLS | Encryption | [Live](https://jalapeno-status.litprotocol.com/) | V1, V2 | Deprecated.  Do not build new apps that use this network. | n/a |
-| Habanero | Decentralized mainnet. Persistent, keys will not be deleted. | BLS, ECDSA | Encryption, User Wallets (PKPs), Serverless Signing (Lit Actions) | Live | V4+ | Good to use | [habanero](https://github.com/LIT-Protocol/networks/tree/main/habanero) | 
+| Habanero | Decentralized mainnet. Persistent, keys will not be deleted. | BLS, ECDSA | Encryption, User Wallets (PKPs), Serverless Signing (Lit Actions) | Live | V4+ | Deprecated.  Do not build new apps that use this network. | [habanero](https://github.com/LIT-Protocol/networks/tree/main/habanero) | 
+Datil-prod | Decentralized mainnet. Persistent, keys will not be deleted. | BLS, ECDSA | Encryption, User Wallets (PKPs), Serverless Signing (Lit Actions), Wrapped Keys | Live | V6+ | Good to use | [datil-prod](https://github.com/LIT-Protocol/networks/tree/main/datil-prod) |
```

Comment:
also, datil-prod is highlighted in red when i check the deploy preview. Let's make sure this is listed first and is in green highlight

Reply:
Good point. I had noticed this but forgot to make a change, thanks